requirements.txt

```diff
@@ -1,4 +1,2 @@
-django==1.3
+django==1.9
```

Comment:
We should make sure it will work on App Engine with that version (i.e. if we're updating the tests, then we should also update the deployed version). In the past, upgrading Django has broken things.